src/pybind/mgr/feedback/controllers/feedback.py

```diff
@@ -0,0 +1,60 @@
+# # -*- coding: utf-8 -*-
+
+from ...dashboard.exceptions import DashboardException
+from ..model.feedback import Feedback
+from ...dashboard.rest_client import RequestException
+from ...dashboard.security import Scope
+from ..services.feedback import CephTrackerClient
+from ...dashboard.controllers import ApiController, ControllerDoc, RESTController
```

Comment:
on it

Reply:
I have removed the irrelevant commented out code.

For the feedback module, I have extended the `MgrModule` and use `store_key` to store the access key. In `mgr.remote` method I can call the method but can't pass the access key as an argument (which is what I'm doing in CLI) without importing the client. Am I understanding it correctly? 

I'm using the `send_command` for now. I have gone through the documentation and few code snippets. Hopefully this should work.

Reply:
Thanks @cypherean ! I'd need to see the code to follow it.

Reply:
hi, I've done this. Please review the code. Right now the feedback module has CLI + tracker client, and controller is in the dashboard module. I have also removed the feedback service and model from dashboard. Updated CLI commands are in the description.

I'm working on:
1. Adding API Key in frontend
2. Updated organised modal after issue creation
3. Writing tests 

Query: If dashboard is enabled but feedback isn't it will give errors. What can I use to enable module before issue creation?

Reply:
@cypherean Maybe disable the Create Issue button and give a hint which has a link to the `Cluster -> Manager Modules -> Feedback module` and then user has to just click enable to enable that module?

Reply:
@epuertat is this the list of always-on modules?